eventually/event/models.py

```diff
@@ -101,34 +124,34 @@ def to_dict(self):
 
         | {
         |    'id': 4,
+        |    'team': 23,
+        |    'owner': 12
         |    'name': 'example',
-        |    'description': 'decsp',
+        |    'description': 'description',
         |    'start_at': 1509539867,
         |    'created_at': 1509539867,
         |    'updated_at': 1509539867,
         |    'longitude': 7.03125,
         |    'latitude': 21.105,
         |    'budget': 100,
         |    'status': 0,
-        |    'team': 23,
-        |    'owner': 12
         | }
         """
 
         return {
             'id': self.id,
+            'team': self.team.id,
```

Comment:
you have team = models.ForeignKey(CustomUser, null=True)
so add self.team.id if self.owner else None,

Reply:
team id is necessary so we do not need `if` statement in this line